### DISCUSSION TITLE
ipv6 multicast routing on overlay network?

### DISCUSSION THREAD

**Discussion by khimaros** [yggdrasil-network/yggdrasil-go#991] [Q&A]
does yggdrasil support global scope multicast routing on the overlay network?

the developer of [reticulum](https://github.com/markqvist/Reticulum) @markqvist and i were discussing the potential of IPv6 multicast peer autodiscovery over yggdrasil but i couldn't find design or current behavior documented anywhere. thank you for the help!

**Comment by neilalexander** [comment]
Yggdrasil does not do any routing of multicast traffic at the moment, largely due to the fact that we don’t yet have a good way of sharing group memberships in a way that actually scales.

**Reply by markqvist** [reply]
Thanks a lot for the feedback.

Does Yggdrasil currently offer any sort of functionality that would be suitable for globally discovering peers, without resorting to the use of a centralised server, a DHT with bootstrap nodes, or similar? As in, an efficient way to discover the Yggdrasil unicast IPv6 address for a given service or similar? It would be very useful for adding Yggdrasil support to Reticulum.

Sorry that I am probably being ignorant here, but I know very little about the under-the-hood specifics of Yggdrasil right now ;)
